compareECLFiles.cmake

```diff
@@ -580,20 +608,48 @@ add_test_compareECLFiles(CASENAME 9_4a_grpctl_msw_model2
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_4b_grpctl_stw_model2
+                         FILENAME 9_4B_WINJ_VREP-W_STW
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
 add_test_compareECLFiles(CASENAME 9_4b_grpctl_msw_model2
                          FILENAME 9_4B_WINJ_VREP-W_MSW
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_4c_grpctl_stw_model2
+                         FILENAME 9_4C_WINJ_GINJ_VREP-W_REIN-G_STW
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
 add_test_compareECLFiles(CASENAME 9_4c_grpctl_msw_model2
                          FILENAME 9_4C_WINJ_GINJ_VREP-W_REIN-G_MSW
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_4d_grpctl_stw_model2
+                         FILENAME 9__4D_WINJ_GINJ_GAS_EXPORT_STW
```

Comment:
I believe the `FILENAME` here should be
```
9_4D_WINJ_GINJ_GAS_EXPORT_STW
```
With a single underscore (`_`) between the leading `9` and `4D`.